### PR TITLE
Documented getAuditTime, getPayload methods in AuditEntry.java

### PR DESCRIPTION
**Pull request by SainathB**
Documented getAuditTime, getPayload methods in AuditEntry.java

### CODE REVIEW

**Comment by fjy** [comment]
@SainathB Thanks for the contrib! Do you mind removig the @author tag, we typically don't have that.

Also, can you fill out our CLA? http://druid.io/community/cla.html

File: common/src/main/java/io/druid/audit/AuditEntry.java [hunk 1]
```diff
@@ -72,12 +72,21 @@ public AuditInfo getAuditInfo()
     return auditInfo;
   }
 
+  /**
+  * @author gianm
```
Comment (nishantmonu51) [inline common/src/main/java/io/druid/audit/AuditEntry.java]: we do not use author tags in code. please remove

File: common/src/main/java/io/druid/audit/AuditEntry.java [hunk 2]
```diff
@@ -72,12 +72,21 @@ public AuditInfo getAuditInfo()
...
+  * @return returns payload as String
+  */
   @JsonProperty
   public String getPayload()
   {
     return payload;
   }
-
+  /**
+  * @author gianm
```
Comment (nishantmonu51) [inline common/src/main/java/io/druid/audit/AuditEntry.java]: please remove author tag. See https://github.com/druid-io/druid/blob/master/CONTRIBUTING.md

**Comment by SainathB** [comment]
Hi, I filled CLA form and removed author tag.

**Pull request merged**